modules/installation-guide/partials/proc_manually-upgrading-che-in-operatorhub.adoc

```diff
@@ -3,32 +3,38 @@
 // upgrading-che-using-operatorhub
 
 [id="manually-upgrading-che-in-operatorhub_{context}"]
-
 = Manually upgrading {prod-short} in OperatorHub
+
+OperatorHub is an assembly point for sharing Operators. The OperatorHub helps you deploy and update applications. The following section describes the process of upgrading {prod-short} by using OperatorHub and the `Manual` approval strategy approach. Use the `Manual` approval strategy to prevent automatic updates of the Operator with every release.
+
 .Prerequisites
 
-* An administrator account on an OpenShift instance.
+* An administrator account on an instance of OpenShift.
 
-* An instance of an earlier minor version of {prod-short}, installed using the Operator from OperatorHub on the same instance of OpenShift.
+* An instance of an earlier minor version of {prod-short}. You have installed the instance by using the Operator.
```

Comment:
`from OperatorHub` is important

Reply:
```suggestion
* An instance of an earlier minor version of {prod-short} that you installed by using OperatorHub.
```
IBM SG:
```
that conjunction
Use "that", without a comma, to introduce a restrictive clause. Use "which", preceded by a comma, to introduce a
nonrestrictive clause.
Consider the following examples: (1) The system units, which have two drives, are floor-standing models. (Nonrestrictive) (2)
The system units that have two drives are floor-standing models. (Restrictive) In the first example, all of the system units are
floor-standing models and each has two drives. The second example implies that some of the system units have two drives
and some do not; the sentence refers only to system units with two drives. See the IBM Style topics Commas and Pronouns for
more information.
```

Reply:
If the reader isn't the person who installed the "earlier minor version of {prod-short}," how can they determine whether it was installed by using OperatorHub versus some other method?

Reply:
@rolfedh 
What about:

1) An existing earlier instance of {prod-short} installed from OperatorHub. 

2) An existing instance of {prod-short} {prod-prev-ver} or older, installed by OperatorHub.

cc @tolusha

Reply:
How about: "An instance of {prod-short} {prod-prev-ver} or earlier that was installed by using OperatorHub."
- We can drop "existing" as redundant.
- IBM SG recommends using "earlier" or "later" to describe versions.

Reply:
IBM SG
```
References to earlier or later versions
To state a requirement that includes earlier or later versions of a product, use the phrases or earlier to indicate earlier versions
and or later to indicate subsequent versions.
To use the transformers, you need DB2 Universal Database 5.1 or later.
This memo applies to IBM Mobile Connect 3.2 or earlier.
Install DB2 Database for Linux, UNIX, and Windows 9.7 or later.
```

Reply:
@tolusha @rolfedh @MichalMaler it's fixed. Thank you so much for your suggestions!